pkg/common/client.go

```diff
@@ -125,6 +127,31 @@ func newTLSConfig(clientCert, clientKey, caCert string) (*tls.Config, error) {
 	return config, nil
 }
 
+func MakeAdminClient(clientID string, kafkaAuthCfg *utils.KafkaAuthConfig, bootstrapServers []string) (sarama.ClusterAdmin, error) {
+	saramaConf := sarama.NewConfig()
+	saramaConf.Version = sarama.V1_1_0_0
+	saramaConf.ClientID = clientID
+
```

Comment:
The distributed implementation already has a utility for creating different types (kafka, eventhubs, custom) of ClusterAdmins - is there any way to use/share that?   If we we're to try and consolidate on this new/common version we would have to port all that type specific (and interface wrapper) implementation over anyway (Since we need/use it)